docs/wiki/Header.md

```diff
@@ -0,0 +1,76 @@
+<a id="top"></a>
+
+_Last updated for Bison Wallet v1.0.0._
+
+**Contents**
+
+- [Notifications Panel](#notifications-panel)
+  - [Notifications Tab](#notifications-tab)
+  - [Recent Activity Tab](#recent-activity-tab)
+- [Language Switching](#language-switching)
+- [Settings Panel](#settings-panel)
+
+
+This page is part of the [Using Bison Wallet](Using-Bison-Wallet) guide, it assumes you have already 
+set up a Bison Wallet using the [Getting Started Guide](Getting-Started).
+
+The Header is displayed at the top of the window, and it is used to access 
+the different features Bison Wallet offers.
+
+<img src="./images/using-bison-wallet/header.png" width="1200" alt="">
+
+---
+
+# Notifications Panel
+
+The notifications' panel can be opened by clicking the notifications icon
+on the [header](#header).
+
+<img src="./images/using-bison-wallet/header-notifications.png" width="400" alt="">
+
+## Notifications Tab
+
+The notifications tab displays important updates, alerts, and other actionable information related 
+your wallets, transaction and DCRDEX accounts.
+
+<img src="./images/using-bison-wallet/notifications-note.png" width="400" alt="">
+
+## Recent Activity Tab
+
+The Recent Activity tab displays lower severity notifications that are not persistent across sessions. 
+
+<img src="./images/using-bison-wallet/notifications-poke.png" width="400" alt="">
+
+# Language Switching
+
+Language Switching can be accessed by clicking the hamburger menu in the [header](#header).
+
+<img src="./images/using-bison-wallet/header-hamburger.png" width="400" alt="">
+
+The first line indicates the current language, while the second has the different options for
+selection.
+
+<img src="./images/using-bison-wallet/header-language.png" width="300" alt="">
```

Comment:
We now have icons for language

![image](https://github.com/user-attachments/assets/209f4fce-1153-4066-8972-1573193c37f5)

Reply:
didn't see this on the tagged v1.0.2

![image](https://github.com/user-attachments/assets/be0386de-d5a9-48d5-82f0-d9f9b33a7bd4)

Reply:
I think commit `a700197c90d42fdf2ef2f983ddec1a162824af7b` is where the text was changed for national flag images - in `client/webserver/site/src/js/app.ts`, I could be wrong.

![image](https://github.com/user-attachments/assets/cfdf0e86-3686-4ce6-ab73-ef0c7ac9848e)


```js
const languageData: Record<string, LangData> = {
  'en-US': {
    name: 'English',
    flag: '🇺🇸' // Not 🇬🇧. MURICA!
  },
  .
  .
```